topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.md

```diff
@@ -130,25 +154,11 @@ The first tool in the workflow is the [**HISAT2**](http://ccb.jhu.edu/software/h
 
 ![Variant calling](../../images/variant_calling.png)
 
-Provided with some BAM dataset(s) and a reference sequence, FreeBayes will generate
-a VCF dataset describing SNPs, indels, and complex variants in samples in the input
-alignments. By default, FreeBayes will consider variants supported by at least two observations
-in a single sample (`-C`) and also by at least 20% of the reads from a single sample (`-F`). These
-settings are suitable for low to high depth sequencing in haploid and diploid samples, but
-users working with polyploid or pooled samples may adjust them depending on the characteristics
-of their sequencing data.
+Provided with some BAM dataset(s) and a reference sequence, FreeBayes will generate a VCF dataset describing SNPs, indels, and complex variants in samples in the input alignments. By default, FreeBayes will consider variants supported by at least two observations in a single sample (`-C`) and also by at least 20% of the reads from a single sample (`-F`). These settings are suitable for low to high depth sequencing in haploid and diploid samples, but users working with polyploid or pooled samples may adjust them depending on the characteristics of their sequencing data.
 
-FreeBayes is capable of calling variant haplotypes shorter than a read length where multiple
-polymorphisms segregate on the same read. The maximum distance between polymorphisms phased in
-this way is determined by the `--max-complex-gap`, which defaults to 3bp. In practice, this can
-comfortably be set to half the read length. Ploidy may be set to any level (`-p`), but by default
-all samples are assumed to be diploid. FreeBayes can model per-sample and per-region variation
-in copy-number (`-A`) using a copy-number variation map.
+FreeBayes is capable of calling variant haplotypes shorter than a read length where multiple polymorphisms segregate on the same read. The maximum distance between polymorphisms phased in this way is determined by the `--max-complex-gap`, which defaults to 3bp. In practice, this can comfortably be set to half the read length. Ploidy may be set to any level (`-p`), but by default all samples are assumed to be diploid. FreeBayes can model per-sample and per-region variation in copy-number (`-A`) using a copy-number variation map.
 
-FreeBayes can act as a frequency-based pooled caller and can describe variants and haplotypes in
-terms of observation frequency rather than called genotypes. To do so, use --pooled-continuous
-and set input filters to a suitable level. Allele observation counts will be described by AO
-and RO fields in the VCF output.
+FreeBayes can act as a frequency-based pooled caller and can describe variants and haplotypes in terms of observation frequency rather than called genotypes. To do so, use --pooled-continuous and set input filters to a suitable level. Allele observation counts will be described by AO and RO fields in the VCF output.
```

Comment:
```suggestion
FreeBayes can act as a frequency-based pooled caller and can describe variants and haplotypes in terms of observation frequency rather than called genotypes. To do so, use `--pooled-continuous` and set input filters to a suitable level. Allele observation counts will be described by AO and RO fields in the VCF output.
```